src/main/java/com/applicaster/sport1player/Sport1PlayerAdapter.java

```diff
@@ -128,7 +129,7 @@ protected void openLoginPluginIfNeeded(final boolean isInline) {
         final LoginContract loginPlugin = LoginManager.getLoginPlugin();
         this.isInline = isInline;
         if (loginPlugin != null ){
-
+            accessToken = loginPlugin.getToken();
```

Comment:
You should add this call after the login, as at this moment is possible we don´t have yet a token.